versions.yaml

```diff
@@ -88,8 +88,8 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/qemu/qemu"
-      version: "4.1.1"
-      tag: "v4.1.1"
+      version: "5.0.0"
```

Comment:
Do you expect that  both qemu versions will be added in obs, kata-deploy, snap?

Reply:
nop, only 5.0.  4.1.1 will be used in the CI when the kernel version < 5.3.0

Reply:
Ok, as heads up, today we have automated testing for kata-deploy and obs packages in azure, those may be affected.

Reply:
how they can be affected? you mean failures? or just a change in the qemu's version?

Reply:
@devimc If we go this route, it would also mean we would need to package both 4.1 and 5.0 qemu, as someone using Kata on Azure cloud may not be able to do so with the 5.0 qemu packages.

Reply:
@amshinde I think we shouldn't deal with this, since this is bug between QEMU and hyperv, users have two options:
1. update Linux kernel to >=5.3.0
2. build and install QEMU 4.1.1 (if they don't like the first option)

Reply:
@devimc those would be affected in case we don't provide qemu 4.X as default for kata-deploy, or OBS. 
- Start the  CI for OBS/Kata-deploy in azure => CI fails because QEMU 4.0 is not used. 

>    update Linux kernel to >=5.3.0
>    build and install QEMU 4.1.1 (if they don't like the first option)

In this case, we are the users, in case we want to continue updating to 5.x, it needs to choose one of those options and implement it and none of those are really user friendly in my opinion :P